optimum/habana/transformers/generation/utils.py

```diff
@@ -99,14 +99,12 @@
     "starcoder2",
     "persimmon",
     "qwen2",
-    "starcoder2",
     "llava",
     "llava_next",
     "stablelm",
     "mamba",
     "deci",
     "qwen2_moe",
-    "gemma",
```

Comment:
Why removing these models from this list?

Reply:
Those two lines are duplicated. You see that 'startcoder2' is in the list twice. same for 'gemma'
https://github.com/huggingface/optimum-habana/blob/main/optimum/habana/transformers/generation/utils.py#L99

Reply:
Wow great catch!